code/game/gamemodes/changeling/powers/adrenaline.dm

```diff
@@ -10,12 +10,10 @@
 //Recover from stuns.
 /obj/effect/proc_holder/changeling/adrenaline/sting_action(mob/living/user)
 	user << "<span class='notice'>Energy rushes through us.[user.lying ? " We arise." : ""]</span>"
-	user.stat = 0
```

Comment:
There's no reason to manually change stat here anymore, if a stat change is actually needed it will be handled by the Setxxxxxx() below.